components/org.wso2.carbon.appmgt.api/src/main/java/org/wso2/carbon/appmgt/api/APIConsumer.java

```diff
@@ -30,13 +30,13 @@
 public interface APIConsumer extends APIManager {
 
     /**
-     * Returns the details of the owner of the given app.
+     * Returns the details of the owner for a given owner Id.
      *
-     * @param appId
+     * @param businessOwnerId Id of business owner.
```

Comment:
Java doc comments issues and formatting issues are fixed in https://github.com/wso2/carbon-appmgt/pull/539/files